documentation/imagery/example.json

```diff
@@ -0,0 +1,102 @@
+[
```

Comment:
I know this is just an example--we may want to scrub the real URL, or generalize the whole example. I don't think our license allows us to distribute the imagery in this way.